src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        final String dateFormat = "dd.MM.yyyy";
+        final String headMessage = "Report for period ";
+        final DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern(dateFormat);
+        StringBuilder totalSalaryInfo = new StringBuilder();
+        totalSalaryInfo.delete(0, totalSalaryInfo.length());
```

Comment:
why do we need this `totalSalaryInfo.delete(0, totalSalaryInfo.length());`?

Reply:
it was necessary when line:
12  StringBuilder totalSalaryInfo = new StringBuilder(); 
was in class declaration area.

Reply:
deleted